driver-core/src/test/java/com/datastax/driver/core/CCMAccess.java

```diff
@@ -236,4 +243,14 @@
      */
     void waitForDown(int node);
 
+    /**
+     * @return The desired target protocol version based on the 'cassandra.version' System property.
+     */
+    ProtocolVersion getDesiredProtocolVersion();
```

Comment:
I would describe it as the highest version common to the driver and the server, is that accurate?